modules/troubleshooting-enabling-kdump.adoc

```diff
@@ -0,0 +1,48 @@
+// Module included in the following assemblies:
+//
+// * support/troubleshooting/troubleshooting-operating-system-issues.adoc
+
+:_content-type: PROCEDURE
+[id="enabling-kdump"]
+= Enabling kdump
+
+{op-system} ships with `kexec-tools`, but manual configuration is required to enable `kdump`.
+
+.Procedure
+
+Perform the following steps to enable `kdump` on {op-system}.
+
+. To reserve memory for the crash kernel during the first kernel booting, provide kernel arguments by entering the following command:
++
+[source, terminal]
+----
+# rpm-ostree kargs --append='crashkernel=256M'
+----
+
+. Optional: To write the crash dump over the network or to some other location, rather than to the default local `/var/crash` location, edit the `/etc/kdump.conf` configuration file.
++
+[NOTE]
+====
+Network dumps are required when using LUKS. `kdump` does not support local crash dumps on LUKS-encrypted devices.
+====
++
+For details on configuring the `kdump` service, see the comments in `/etc/sysconfig/kdump`, `/etc/kdump.conf`, and the `kdump.conf` manual page.
+ifdef::openshift-enterprise[]
+Also refer to the link:https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/8/html/managing_monitoring_and_updating_the_kernel/configuring-kdump-on-the-command-line_managing-monitoring-and-updating-the-kernel[RHEL `kdump` documentation] for further information on configuring the dump target.
```

Comment:
Maybe spell out "Red Hat Enterprise Linux (RHEL)" here for this first instance, though this is also an existing usage so 🤷.